sdbuild/ubuntu/bionic/aarch64/config

```diff
@@ -1,3 +1,3 @@
-STAGE2_PACKAGES_aarch64 := gcc-mb python_packages_bionic libsds jupyter
+STAGE2_PACKAGES_aarch64 := gcc-mb libsds python_packages_bionic jupyter
```

Comment:
just FYI: libsds provides libxlnk_cma.h which is required by the _xhdmi packages that are built by PYNQ in the python_packages_bionic.

Reply:
`pynq` has its dedicated turn to build; I don't think it is getting built in `python_packages_bionic`?

Reply:
is that guaranteed to be before libsds ?  Im running on a 112 core system vs the Vagrant default. Happy to try it out with this change and piecemeal the changes into master.

Reply:
If you look at the same file, you can see `pynq` gets installed later. So what I am saying is that you don't have to change this part.

Reply:
ok. Will remove in next patch. Looking for guidance on resolv.conf